src/models/hooks/goalFieldResponse.js

```diff
@@ -70,6 +72,32 @@ const syncActivityReportGoalFieldResponses = async (sequelize, instance, _option
       },
     );
 
+    // Get all the ActivityReportGoals that are missing a response.
+    const missingResponses = await sequelize.models.ActivityReportGoal.findAll({
```

Comment:
This code is responsible for creating any missing activity report goal field responses that might have existed before the prompts were added.